src/main/scala/com.snowplowanalytics.snowplow.micro/Configuration.scala

```diff
@@ -124,7 +124,7 @@ object Configuration {
       resolver <- Resolver.fromConfig[IO](resolverConfig).leftMap(_.show)
       completeResolver = resolver.copy(repos = resolver.repos ++ readIgluExtraRegistry())
       client <- EitherT.liftF(IgluCirceClient.fromResolver[IO](completeResolver, resolverConfig.cacheSize))
-    } yield IgluResources(resolver, client)
+    } yield IgluResources(completeResolver, client)
```

Comment:
I sometimes use shadowing in for-comprehensions to avoid that category of bug:

```scala
  resolver <- Resolver.fromConfig[IO](resolverConfig).leftMap(_.show)
  resolver <- EitherT.rightT[IO, String](resolver.copy(repos = resolver.repos ++ readIgluExtraRegistry()))
```